frontend/src/containers/NotificationModal.vue

```diff
@@ -146,6 +144,14 @@ export default {
         this.redirectPage(notif);
       } 
     },
+    formattedDate(timestamp) {
+      var currDate = new Date(timestamp);
+      var numberOfMlSeconds = currDate.getTime();
+      var subFourHours = 4 * 60 * 60 * 1000; //GMT -4
+      var newDate = new Date(numberOfMlSeconds - subFourHours);
+      var newTime = newDate.toString().split(' GMT')[0];
+      return newTime.substring(0, newTime.indexOf(':') +3);
+    },
```

Comment:
This method was created in the `NotificationModal.vue` file. Since it is probable useful in other parts of the code, would it be easier to create it in the utils folder and call it in the various places? Let me know what you think.

Reply:
I have addressed this refactoring in this [commit](https://github.com/purplesmurf1998/Autotracks/pull/105/commits/4016c674218b60f0932c893a6c8dcec14266843c)